polaris-intellisense/.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+// Place your settings in this file to overwrite default and user settings.
+{
+    "files.exclude": {
+        "out": false // set this to true to hide the "out" folder with the compiled JS files
+    },
+    "search.exclude": {
+        "out": true // set this to false to include "out" folder in search results
+    },
```

Comment:
`out` is a part of the `.gitignore` is there a reason it's a problem to show this folder? It should be excluded in search.

Reply:
We recently moved away from this [convention of hiding files](https://github.com/Shopify/polaris-react/pull/5187/files) in `polaris-react` in favor of the VSCode defaults. I support this decision and feel we shouldn't have an opinion here.